bin/bundle

```diff
@@ -14,6 +14,14 @@ $LOAD_PATH.each do |path|
   end
 end
 
+# Require all the user level plugins
+Bundler.settings.plugins.each do |plugin|
```

Comment:
doing this in the exe is going to be potentially very slow, is there any way we could avoid loading plugins when not 100% necessary?